test/bin/mdx-dune-gen/misc/non-deterministic/dune

```diff
@@ -0,0 +1,52 @@
+(rule
+ (target dune_gen.ml)
+ (action
+  (with-stdout-to
+   %{target}
+   (run ocaml-mdx dune-gen))))
+
+(executable
+ (name dune_gen)
+ (modules dune_gen)
+ (modes byte)
+ (libraries mdx.test))
+
+(rule
+ (target dune-mdx-nondeterministic.deterministic)
+ (deps
+  dune_gen.exe
+  (:input dune-mdx-nondeterministic))
+ (action
+  (with-stdout-to
+   %{target}
+   (run ./dune_gen.exe %{input}))))
```

Comment:
same here, you can inline input and target and use the short form with just the action.